guides/v3.7.0/components/defining-a-component.md

```diff
@@ -73,14 +67,14 @@ file at `app/components/blog-post.js`. If your component was called
 ## Dynamically rendering a component
```

Comment:
Should probably also mention

```hbs
{{#let (component (concat 'some-' this.thing)) as |SomeThing|}}
  <SomeThing/>
{{/let}}
```

Reply:
how about this:

The real value of [`{{component}}`](https://www.emberjs.com/api/ember/release/classes/Ember.Templates.helpers/methods/component?anchor=component) comes from being able to dynamically pick
the component being rendered. Below is an example of using the helper as a
means of choosing different components for displaying different kinds of posts:

```handlebars {data-filename=app/templates/components/foo-thing.hbs}
<h3>Hello from foo!</h3>
<p>{{this.post.body}}</p>
```

```handlebars {data-filename=app/templates/components/bar-thing.hbs}
<h3>Hello from bar!</h3>
<div>{{this.post.author}}</div>
```

```javascript {data-filename=app/routes/index.js}
import Route from '@ember/routing/route';

export default Route.extend({
  model() {
    return this.store.findAll('post');
  }
});
```

```handlebars {data-filename=app/templates/index.hbs}
{{#each this.model as |post|}}
  {{!-- either foo-thing or bar-thing --}}
  {{component post.componentName post=post}}
{{/each}}
```

or 

```handlebars {data-filename=app/templates/index.hbs}
{{#each this.model as |post|}}
  {{!-- either foo-thing or bar-thing --}}
  {{#let (component (concat this.componentName '-thing')) as |SomeThing|}}
    <SomeThing />
  {{/let}}
{{/each}}
```

Reply:
I'd keep the second example almost the same:

```hbs
{{#each this.model as |post|}}
  {{!-- either foo-thing or bar-thing --}}
  {{#let (component (concat this.componentName)) as |Post|}}
    <Post @post={{post}} />
  {{/let}}
{{/each}}
```